### PR TITLE
Added Ruby support for el7

### PR DESCRIPTION
**Pull request by neilmassey**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -154,10 +154,17 @@ set_facter() {
...
-    yum install -y https://s3-eu-west-1.amazonaws.com/msm-public-repo/ruby/ruby-2.1.5-2.el6.x86_64.rpm
+  majorversion=$(lsb_release -rs | cut -f1 -d.)
+  if [[ "$majorversion" == "6" ]]; then
+  echo "Linux Major Version 6"
+   ruby -v  > /dev/null 2>&1
+   if [[ $? -ne 0 ]] || [[ $(ruby -v | awk '{print $2}' | cut -d '.' -f 1) -lt 2 ]]; then
+     yum remove -y ruby-*
+     yum install -y https://s3-eu-west-1.amazonaws.com/msm-public-repo/ruby/ruby-2.1.5-2.el6.x86_64.rpm
+   fi
+ elif [[ "$majorversion" == "7" ]]; then
```
Comment (seanclerkin) [inline init.sh]: Indentation

File: init.sh [hunk 2]
```diff
@@ -154,10 +154,17 @@ set_facter() {
...
+  if [[ "$majorversion" == "6" ]]; then
+  echo "Linux Major Version 6"
+   ruby -v  > /dev/null 2>&1
+   if [[ $? -ne 0 ]] || [[ $(ruby -v | awk '{print $2}' | cut -d '.' -f 1) -lt 2 ]]; then
+     yum remove -y ruby-*
+     yum install -y https://s3-eu-west-1.amazonaws.com/msm-public-repo/ruby/ruby-2.1.5-2.el6.x86_64.rpm
+   fi
+  elif [[ "$majorversion" == "7" ]]; then
+    echo "Linux Major version 7"
+    yum_install ruby
```
Comment (chrisjomaron) [inline init.sh]: are you happy that the yum provided ruby will always be OK, or would you want to fix it at a specific version, similar to RHEL6/CentOS6?

Comment (seanclerkin) [inline init.sh]: Your point is valid although our requirements are only >2.0.0 at the moment so I think we can ride the version wave until we hit a problem.

**Comment by chrisjomaron** [comment]
:+1:

**Comment by seanclerkin** [comment]
:+1:

**Pull request merged**